lib/runtime/wasmer/instance.go

```diff
@@ -196,96 +257,28 @@ func GetRuntimeVersion(code []byte) (version runtime.Version, err error) {
 	return version, nil
 }
 
-var (
-	ErrCodeEmpty      = errors.New("code is empty")
-	ErrWASMDecompress = errors.New("wasm decompression failed")
-)
-
-func setupVM(code []byte) (instance wasm.Instance,
-	allocator *runtime.FreeingBumpHeapAllocator, err error) {
-	if len(code) == 0 {
-		return instance, nil, ErrCodeEmpty
-	}
-
-	code, err = decompressWasm(code)
-	if err != nil {
-		// Note the sentinel error is wrapped here since the ztsd Go library
-		// does not return any exported sentinel errors.
-		return instance, nil, fmt.Errorf("%w: %s", ErrWASMDecompress, err)
-	}
-
-	imports, err := importsNodeRuntime()
-	if err != nil {
-		return instance, nil, fmt.Errorf("creating node runtime imports: %w", err)
-	}
-
-	// Provide importable memory for newer runtimes
-	// TODO: determine memory descriptor size that the runtime wants from the wasm.
-	// should be doable w/ wasmer 1.0.0. (#1268)
-	memory, err := wasm.NewMemory(23, 0)
-	if err != nil {
-		return instance, nil, fmt.Errorf("creating web assembly memory: %w", err)
-	}
-
-	_, err = imports.AppendMemory("memory", memory)
-	if err != nil {
-		return instance, nil, fmt.Errorf("appending memory to imports: %w", err)
+// UpdateRuntimeCode updates the runtime instance to run the given code
+func (in *Instance) UpdateRuntimeCode(code []byte) error {
```

Comment:
This function appears to never be called. Delete?

Reply:
We usually create a new instance for runtime upgrades?

Reply:
Yeah exactly